kubernetes-client-openapi/src/main/java/io/micronaut/kubernetes/client/openapi/ssl/DefaultPrivateKeyLoader.java

```diff
@@ -35,7 +36,8 @@
  */
 @Singleton
 @BootstrapContextCompatible
-class DefaultPrivateKeyLoader implements KubernetesPrivateKeyLoader {
+@Internal
+public class DefaultPrivateKeyLoader implements KubernetesPrivateKeyLoader {
```

Comment:
is this public for a reason? If so document why

Reply:
If it is not public it can't be replaced with custom implementation
```
@Replaces(DefaultPrivateKeyLoader.class)
```

Reply:
Documented